### ISSUE TITLE
Looking to add a Simplified RM42C

### ISSUE THREAD

**Issue by atmosphericpro** [bitfocus/companion-module-requests#1232]
Looking to add a Simplified RM42C to be able to change the inputs and outputs for their matrix. How would I go about requesting that?

**Comment by istnv** [comment]
You just did. Please provide a link to the documentation. Can the unit be controlled via ethernet?

**Comment by atmosphericpro** [comment]
Yes of Course. I have a link to the manual and towards the end of the manual is the code parameters
https://acrobat.adobe.com/link/track?uri=urn:aaid:scds:US:8c495e6f-0f02-4512-97bc-e5b101699a30
